src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -25,6 +25,19 @@ public void DumpRuntimeInformationToConsole()
             Console.WriteLine($@"{dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd}");
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Unix interop")]
+        public void DumpRuntimeInformationToConsole2()
+        {
+            // Not really a test, but useful to dump to the log to
+            // sanity check that the test run or CI job
+            // was actually run on the OS that it claims to be on
+            string lcr = PlatformDetection.LibcRelease;
+            string lcv = PlatformDetection.LibcVersion;
+
+            Console.WriteLine($@"LibcRelease={lcr} LibcVersion={lcv}");
```

Comment:
How about just adding this to the DumpRuntimeInformationToConsole test guarded by an IsFullFramework test?  That way all of the information will be clumped together in the log.